src/inference/src/dev/isync_infer_request.cpp

```diff
@@ -92,6 +92,12 @@ void check_batched_tensors(const ov::Output<const ov::Node>& input,
 
 ov::IInferRequest::~IInferRequest() = default;
 
+void ov::IInferRequest::reset_state(){
```

Comment:
Shouldn't we override `reset_state()` in `npuw::LLMInferRequest` or what's the point of these changes?

Reply:
These changes will be dropped